molgenis-navigator/src/main/java/org/molgenis/navigator/download/exception/DownloadFailedException.java

```diff
@@ -0,0 +1,23 @@
+package org.molgenis.navigator.download.exception;
+
+import org.molgenis.i18n.CodedRuntimeException;
+
+public class DownloadFailedException extends CodedRuntimeException {
+  private static final String ERROR_CODE = "NAV01";
```

Comment:
```suggestion
  private static final String ERROR_CODE = "N03";
```
To harmonize with Copy Exceptions

Reply:
harmonizing seems like a good plan, I propose to "NAV##". I find the three letters clearer, and one-lettr is not feasable for all modules anyway